lib/platform/gitlab/index.ts

```diff
@@ -397,11 +397,8 @@ async function fetchPrList(): Promise<Pr[]> {
   const searchParams = {
     per_page: '100',
   } as any;
-  // istanbul ignore if
-  if (config.ignorePrAuthor) {
-    // https://docs.gitlab.com/ee/api/merge_requests.html#list-merge-requests
-    // default: `scope=created_by_me`
-    searchParams.scope = 'all';
```

Comment:
According to this comment, the api should only return my PR by default.

Can you verify that there is a bug in gitlab api / docs?

Reply:
This is the mixup I mentioned in the description - there's no issue with the Gitlab docs.

The comment in the code is referencing the documentation for the `GET /merge_requests` which does fetch my MRs by default. However Renovate uses the `GET /projects/:id/merge_requests` endpoint which fetches everyone's MRs for a project and is documented here: https://docs.gitlab.com/ee/api/merge_requests.html#list-project-merge-requests.

Reply:
Here's an example of the endpoint returning all MRs by default using Gitlab's public repo: https://gitlab.com/api/v4/projects/278964/merge_requests/

Passing an `author_id` filters the list: https://gitlab.com/api/v4/projects/278964/merge_requests/?author_id=3468028